packages/inventories-exporter/test/main.spec.js

```diff
@@ -149,11 +148,7 @@ describe('InventoryExporter', () => {
           return Promise.resolve()
         })
       const outputStream = streamtest.v2.toText((error, result) => {
-        const expectedResult = stripIndent`
-          sku,quantityOnStock,restockableInDays
-          hello,me,4
-        `
-        expect(result).toEqual(expectedResult)
+        expect(result).toMatchSnapshot()
```

Comment:
why is this snapshotted, and tested differently from `inventories-exporter.it.js`?

Reply:
Good point, fixed in https://github.com/commercetools/nodejs/commit/88cf2beb30cc3dec84abec280fb895ae538ea590